### ISSUE TITLE
Syria full map - Map-baked factories' destruction not tracked (changed object IDs?)

### ISSUE THREAD

**Issue by MaHuJa** [dcs-liberation/dcs_liberation#1308]
**Describe the bug**

Upon successfully destroying factories, they are not registered as destroyed in the liberation app.
This does not apply to liberation-placed objectives (oil rigs etc) which are still destroyed just fine.

Probable cause is the updated syria map, adding cyprus, changing the IDs of all map baked objects.

Fixing this will likely break dcs stable until it is updated, comparable factions including the new hind.

**To Reproduce**

1. Load liberation, 3.0, develop_3.x, develop. (develop-4.x untested, presumably still a problem)
2. Create new campaign using syria full map
3. Plot mission to strike factories, for example the three buildings just south of bassel al-assad, or the NDB beacon tower north of it.
4. Using current OB with the changed syria map
5. Destroy the target, finish the mission

**Expected behavior**

Upon entering the planning stage for the next turn, the destroyed factories should be removed from the map.

**Additional information**

**Version information (please complete the following information):**
 - DCS Liberation [e.g. 2.3.1]:

**Additional context**
Other syria campaigns should probably also be checked.

**Comment by DanAlbert** [comment]
IMO if map object IDs change between DCS releases this feature is not viable. Every campaign breaking whenever DCS updates is unacceptable.

**Comment by MaHuJa** [comment]
Map updates that break IDs don't seem common in the first place.  I think maybe two in the year since Syria was released? Not that I've tracked it closely. 

Even if that is unacceptable, the feature itself is plenty viable on maps that don't see updates - when was NTTR last changed?

Then it's just a matter for campaign creators/maintainers to not use the feature on not-yet-stable maps, or deal with the updating when it happens. We knew this change (filling in a region, cyprus) was going to happen for many months.
